conf/drupal/config/field.storage.paragraph.field_iframe_height_config.yml

```diff
@@ -0,0 +1,27 @@
+uuid: 409932e8-53de-45b6-ac81-308168ebe40a
+langcode: en
+status: true
+dependencies:
+  module:
+    - options
+    - paragraphs
+id: paragraph.field_iframe_height_config
+field_name: field_iframe_height_config
+entity_type: paragraph
+type: list_string
+settings:
+  allowed_values:
+    -
+      value: fixed
+      label: 'Fixed height in pixels (default)'
+    -
+      value: '100'
+      label: '100% (as much height as necessary to show the whole iframe without scrolling within the frame). <strong>If you want the iframe height to adjust based on the content in your iframe, choose 100% and add <a href="/themes/custom/mass_theme/overrides/js/iframe_resizer_iframe.js" target="_blank">this code</a> to your iframed content.</strong>'
```

Comment:
```suggestion
      label: '100% (as much height as necessary to show the whole iframe without scrolling within the frame). <strong>If you want the iframe height to adjust based on the content in your iframe, choose 100% and add the following script into your iframe HTML source code before the closing body tag:
<code>"<script type="text/javascript" src="https://mass.gov/themes/custom/mass_theme/overrides/js/iframe_resizer_iframe.js"></script>"<code>.</strong>'
```

Reply:
I think it will be more straightforward to ask the authors to add the CDN version